testsuite/features/step_definitions/command_steps.rb

```diff
@@ -324,16 +324,12 @@
   raise "Download debug file failed" unless code.zero?
 end
 
-Then(/^I get logfiles from "([^"]*)"$/) do |target|
-  node = get_target(target)
-  os_version, os_family = get_os_version(node)
-  if os_family =~ /^opensuse/
-    node.run('zypper mr --enable os_pool_repo os_update_repo && zypper --non-interactive install tar')
-  end
-  node.run("journalctl > /var/log/messages && (tar cfvJP /tmp/#{target}-logs.tar.xz /var/log/ || [[ $? -eq 1 ]])")
-  `mkdir logs` unless Dir.exist?('logs')
-  code = file_extract(node, "/tmp/#{target}-logs.tar.xz", "logs/#{target}-logs.tar.xz")
-  raise "Download log archive failed" unless code.zero?
+When(/^I extract the log files from all our active nodes$/) do
+  $nodes.each do |node|
```

Comment:
I don't think we have used until now a global variable `$nodes`.
This PR gives a good point to add one.

Reply:
Thank you, indeed, missed.